webui/react/src/stores/cluster.tsx

```diff
@@ -28,12 +28,21 @@ const initClusterOverview: ClusterOverview = {
   [ResourceType.UNSPECIFIED]: structuredClone(initResourceTally),
 };
 
+const flexSchedulers: V1SchedulerType[] = [V1SchedulerType.PBS, V1SchedulerType.SLURM];
```

Comment:
nit: prob use `Set`? or directly compare these in if statement since there are only 2 values.

Reply:
✅ redid as Set